### PR TITLE
Modify error messages for rdel and sdel commands

### PR DESCRIPTION
**Pull request by jamietan2002**

### CODE REVIEW

File: src/main/java/foodwhere/commons/core/Messages.java [hunk 1]
```diff
@@ -6,9 +6,10 @@
 public class Messages {
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
-    public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
+    public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! %1$s";
     public static final String MESSAGE_INVALID_STALL_DISPLAYED_INDEX = "The stall index provided is invalid";
     public static final String MESSAGE_INVALID_REVIEW_DISPLAYED_INDEX = "The review index provided is invalid";
+    public static final String MESSAGE_INVALID_INDEX = "Index is not a single non-zero unsigned integer.\n";
```
Comment (clarence-chew) [inline src/main/java/foodwhere/commons/core/Messages.java:12]: This seems a bit too technical for the user, talking about "unsigned integer" - how about "The index cannot be read as a positive integer."?

File: src/test/java/foodwhere/logic/parser/ParserUtilTest.java [hunk 2]
```diff
@@ -37,7 +38,7 @@ public void parseIndex_invalidInput_throwsParseException() {
 
     @Test
     public void parseIndex_outOfRangeInput_throwsParseException() {
```
Comment (clarence-chew) [inline src/test/java/foodwhere/logic/parser/ParserUtilTest.java:40]: I think this needs some test cases with whitespace to be trimmed, like `WHITESPACE + "1" + WHITESPACE`.

**Review by clarence-chew — COMMENTED** [review]
I have some comments.

**Review by clarence-chew — APPROVED** [review]
LGTM!
